src/app/features/courses/courseCard.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from "@angular/core";
+
+
+@Component({
+    selector: 'card',
+    templateUrl: './courseCard.component.html',
+    styleUrls: ['./courseCard.component.css']
+
+})
+
+export class CourseCardComponent {
+
```

Comment:
please pay attention to formatting, you could check the channel, I provided the link to initial repo with configured formating , please use it as a basis for your tasks